src/simplestyle.js

```diff
@@ -38,27 +38,29 @@ function remap(a) {
     return d;
 }
 
-
 export function style(feature) {
     return remap(fallback(feature.properties || {}, defaults));
 }
 
-
 // Pinched from mapbox.js
 // https://github.com/mapbox/mapbox.js/blob/publisher-production/src/marker.js
 export function getIcon(fp) {
     fp = fp || {};
     const sizes = {
-            small: [20, 25],
-            medium: [30, 39],
-            large: [35, 45]
+            small: [20, 50],
+            medium: [30, 70],
+            large: [35, 90]
         },
         size = fp['marker-size'] || 'medium',
+        // see valid symbols https://labs.mapbox.com/maki-icons/
+        symbol = ('marker-symbol' in fp && fp['marker-symbol'] !== '') ? `-${  fp['marker-symbol']}` : '',
         color = (fp['marker-color'] || '7e7e7e').replace('#', '');
 
-    const svg = `<svg width='354' height='458' xmlns='http://www.w3.org/2000/svg'><path d='M177 0C79 0 0 79 0 177c0 95 163 270 170 277a9 9 0 0013 0c7-7 170-182 170-277C353 79 274 0 177 0z' fill='#${color}'/></svg>`
     return L.icon({
-        iconUrl: encodeURI(`data:image/svg+xml,${svg}`).replace('#', '%23'),
+        // from http://bl.ocks.org/tmcw/3861338
+        iconUrl: `//a.tiles.mapbox.com/v3/marker/pin-${
```

Comment:
I believe, for some reason, this API does not require a token. As I mentioned it is the same used in geojson.io, so I figure it's ok here?